package.json

```diff
@@ -23,7 +23,7 @@
   },
   "dependencies": {
     "assert": "^1.4.1",
-    "gc-profiler": "^1.4.0"
```

Comment:
I don't have an context on `gc-profiler` vs. `gc-stats`, but if it's faster and stops tests from hanging, I'm all for it!

Reply:
Yeah, exactly. Seems to be faster and fixes our issue with tests.